src/app.js

```diff
@@ -173,20 +130,15 @@ export default function (app) {
         element.style[i] = value[i]
       }
     } else {
-      if (typeof value !== "function") {
-        element.setAttribute(name, value)
-      }
-
-      if (element.namespaceURI !== SVG_NS) {
-        if (element.type === "text") {
-          var oldSelStart = element.selectionStart
-          var oldSelEnd = element.selectionEnd
-        }
-
+      try {
         element[name] = value
+      } catch (_) { }
 
-        if (oldSelStart >= 0) {
-          element.setSelectionRange(oldSelStart, oldSelEnd)
+      if (typeof value !== "function") {
+        if (value) {
```

Comment:
Yields the same amount of bytes because it gets converted to that during minification.

```
n?e.setAttribute(t,n):e.removeAttribute(t)
```